### PR TITLE
add index.md files to topics

### PR DESCRIPTION
**Pull request by mattgreg**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [X] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [X] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [X] Content (and my changes are in `index.md`)

> An index.md file did not exist for either of these pre-existing well populated topics

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by nholden — COMMENTED** [review]
Thanks for this contribution! ❤️ 

What do you think about making an index page for just one of these topics and keeping the other as an alias? It looks like [end-to-end-encryption](https://github.com/topics/end-to-end-encryption) is more popular than [e2ee](https://github.com/topics/e2ee), so I might lean toward choosing that one for the index page.

**Comment by mattgreg** [comment]
💯 

 `end-to-end-encryption` is more popular, and a more likely search vector for projects. `e2ee` is a good alias for this topic. Having both co-exist as separate lists doesn't make much sense to me.

File: topics/e2ee/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+display_name: E2EE
+aliases: end-to-end-encryption
+github_url: https://github.com/topics/e2ee
```
Comment (mattgreg) [inline topics/e2ee/index.md:4]: @nholden What is the structure for the alias if we want to have E2EE point to End-to-End-Encryption?

I assume it's change `github_url` to point to `https://github.com/topics/end-to-end-encryption`

Then delete lines 5-9

?

File: topics/end-to-end-encryption/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+display_name: End-to-End Encryption
+aliases: e2ee
```
Comment (mattgreg) [inline topics/end-to-end-encryption/index.md:3]: I assume that line 2 needs to be deleted?

File: topics/e2ee/index.md (hunk 1 above)
Comment (nholden) [reply topics/e2ee/index.md:4]: My understanding is that you can delete this file and keep `e2ee` as one of the `aliases` in the `end-to-end-encryption` index.

As an example: the [`cli` topic has `command-line-interface` as an alias](https://github.com/github/explore/blob/main/topics/cli/index.md). Both https://github.com/topics/cli and https://github.com/topics/command-line-interface pull metadata from the `cli` index file (there isn't a separate `command-line-interface` index file).

File: topics/end-to-end-encryption/index.md (hunk 2 above)
Comment (nholden) [reply topics/end-to-end-encryption/index.md:3]: I would keep it [per this comment](https://github.com/github/explore/pull/2428#discussion_r670061428).

File: topics/e2ee/index.md (hunk 1 above)
Comment (mattgreg) [reply topics/e2ee/index.md:4]: CLI is a very helpful example. Stand by for the corrections.

**Pull request merged**